fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/RewritePlanner.java

```diff
@@ -54,6 +54,8 @@
     @Nonnull
     private final RecordStoreState recordStoreState;
     @Nullable
+    private SingleExpressionRef<PlannerExpression> root;
```

Comment:
Is this for future use? As near as I can tell, `root` and `currentRoot` are still always the same.

Reply:
No, I think it must be leftover from some refactoring that I did and then un-did. I removed it.